app/src/models/init-models.ts

```diff
@@ -649,6 +652,8 @@ export function initModels(sequelize: Sequelize) {
   });
   User.belongsTo(User, { as: "organization", foreignKey: "organizationId" });
   User.hasMany(User, { as: "users", foreignKey: "organizationId" });
+  User.hasOne(UserFile, { foreignKey: "userId", as: "user" });
```

Comment:
The user should be able to upload multiple files, so this should be `hasMany` instead of `hasOne`.